lnwallet/remotedcrwallet/extended_wallet.go

```diff
@@ -43,3 +46,28 @@ func (b *DcrWallet) RescanWallet(startHeight int32, progress func(height int32)
 		}
 	}
 }
+
+func (b *DcrWallet) GetWalletTransaction(txh chainhash.Hash) (*lnwallet.WalletTransaction, error) {
+	req := &pb.GetTransactionRequest{
+		TransactionHash: txh[:],
+	}
+	res, err := b.wallet.GetTransaction(b.ctx, req)
+	if status.Code(err) == codes.NotFound {
+		return nil, lnwallet.ErrWalletTxNotExist
+	}
+	if err != nil {
+		return nil, err
+	}
+	var bh *chainhash.Hash
+	if res.BlockHash != nil {
```

Comment:
Why would BlockHash be nil? Is it if the tx is not yet mined?

Reply:
Correct.